src/LiquidityPool.sol

```diff
@@ -214,29 +211,29 @@ contract LiquidityPool is Auth, IERC7540 {
     }
 
     /// @inheritdoc IERC4626
-    function maxWithdraw(address operator) external view returns (uint256 maxAssets) {
-        maxAssets = manager.maxWithdraw(address(this), operator);
+    function maxWithdraw(address owner) external view returns (uint256 maxAssets) {
+        maxAssets = manager.maxWithdraw(address(this), owner);
     }
 
     /// @inheritdoc IERC4626
-    /// @notice DOES NOT support operator != msg.sender since shares are already transferred on requestRedeem
-    function withdraw(uint256 assets, address receiver, address operator) external returns (uint256 shares) {
-        require((msg.sender == operator), "LiquidityPool/not-the-operator");
-        shares = manager.withdraw(address(this), assets, receiver, operator);
-        emit Withdraw(msg.sender, receiver, operator, assets, shares);
+    /// @notice DOES NOT support owner != msg.sender since shares are already transferred on requestRedeem
+    function withdraw(uint256 assets, address receiver, address owner) external returns (uint256 shares) {
+        require((msg.sender == owner), "LiquidityPool/not-the-owner");
```

Comment:
I think, here we need to emphasise that `owner` is very context dependent. The `owner` in this function is not necessarily the owner of the shares locked in escrow (passed in `requestRedeem`), but the receiver of the fullfilled order